schema/project-level/project-schema.json

```diff
@@ -2370,6 +2390,200 @@
+          "description": "A short title for this financing arrangement.",
+          "type": "string",
+          "minLength": 1
+        },
+        "description": {
+          "title": "Description",
+          "description": "A short description of this financing arrangement. This may include details of service fees, commitment fees, lender rights, step in arrangements, exchange rate guarantees, and for syndicated loans, details of syndication and any members of the syndicate.",
+          "type": "string",
+          "minLength": 1
+        },
+        "value": {
+          "title": "Value",
+          "description": "The total value of this finance.",
+          "$ref": "#/definitions/Value"
+        },
+        "financingParty": {
+          "title": "Financing organization",
+          "description": "The organization providing this financing arrangement.",
+          "$ref": "#/definitions/OrganizationReference"
+        },
+        "financingPartyType": {
+          "title": "Financing organization type",
+          "description": "The type of the financing organization, from the closed financingPartyType codelist.",
+          "type": "string",
+          "codelist": "financingPartyType.csv",
+          "openCodelist": false,
+          "enum": [
+            "institutional",
+            "commercial",
+            "bilateral",
+            "multilateral",
+            "vendor",
+            "other"
+          ]
+        },
+        "source": {
+          "title": "Funding source",
+          "description": "The source of funds for this financing arrangement. For example, the Green Climate Fund.",
+          "type": "string",
+          "minLength": 1
+        },
+        "assetClass": {
+          "title": "Asset class",
+          "description": "The asset classes used in this financing arrangement, from the closed assetClass codelist.",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "debt",
+              "equity"
+            ],
+            "minLength": 1
+          },
+          "codelist": "assetClass.csv",
+          "openCodelist": false,
+          "minItems": 1,
+          "uniqueItems": true
+        },
+        "type": {
+          "title": "Type",
+          "description": "The type of this financing arrangement, from the closed financingArrangementType codelist.",
+          "type": "string",
+          "codelist": "financingArrangementType.csv",
+          "openCodelist": false,
+          "enum": [
+            "bond",
+            "loan",
+            "hybrid",
+            "shares.listed",
+            "shares.unlisted",
+            "grant",
+            "guarantee"
+          ]
+        },
+        "repaymentPriority": {
+          "title": "Repayment priority",
+          "description": "The order in which the debt in this financing arrangement will be repaid in case of default, from the closed debtRepaymentPriority codelist.",
```

Comment:
```suggestion
          "description": "The order in which the debt in the financing arrangement will be repaid in case of default, using the closed [debtRepaymentPriority](https://standard.open-contracting.org/infrastructure/{{version}}/{{lang}}/reference/codelists/#debtRepaymentPriority) codelist.",
```